### ISSUE TITLE
setup on Digital Ocean Ubuntu 16.04.1 install

### ISSUE THREAD

**Issue by msh100** [linuxserver/docker-openvpnas#8]
I have been unable to get this image working on a fresh Digitalocean Ubuntu 16.04.1 installation. I have created the user `vpn` which has been granted UID and GID `1000`.

```
docker create \
--name=openvpn-as \
-v /home/vpn/config/:/config \
-e PGID=1000 -e PUID=1000 \
-e TZ=Etc/UTC \
-e INTERFACE=eth0 \
--net=host --privileged \
linuxserver/openvpn-as
```

The image does appear to create two interfaces, `as0t0` and `as0t1` however when I try to go to the address which is printed in the terminal I get `ERR_CONNECTION_REFUSED` (this is also not reachable locally).

I have double checked:
 * The interface name
 * Group and user ID

I have been able to get this image running on another server located elsewhere so I do not think I have misunderstood any of the configuration options however that may be a possibility.

**Comment by sparklyballs** [comment]
run the following

docker exec -it openvpn-as bash
^^^ gives you shell inside the container

ifconfig

**Comment by msh100** [comment]
```
root@vpntest:/# ifconfig
as0t0     Link encap:UNSPEC  HWaddr 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00
          inet addr:172.27.224.1  P-t-P:172.27.224.1  Mask:255.255.248.0
          UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:200
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

as0t1     Link encap:UNSPEC  HWaddr 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00
          inet addr:172.27.232.1  P-t-P:172.27.232.1  Mask:255.255.248.0
          UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:200
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

docker0   Link encap:Ethernet  HWaddr 02:42:cd:3a:82:69
          inet addr:172.17.0.1  Bcast:0.0.0.0  Mask:255.255.0.0
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

eth0      Link encap:Ethernet  HWaddr 04:01:4f:1f:57:01
          inet addr:178.62.35.77  Bcast:178.62.63.255  Mask:255.255.192.0
          inet6 addr: fe80::601:4fff:fe1f:5701/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:2367 errors:0 dropped:0 overruns:0 frame:0
          TX packets:2190 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:1301281 (1.3 MB)  TX bytes:371177 (371.1 KB)

lo        Link encap:Local Loopback
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:164 errors:0 dropped:0 overruns:0 frame:0
          TX packets:164 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1
          RX bytes:12040 (12.0 KB)  TX bytes:12040 (12.0 KB)
```

**Comment by sparklyballs** [comment]
anything starting 
172.27----

are docker internal interfaces, don't use those.

is the inet address of eth0 the same as the ip of the host ?

**Comment by msh100** [comment]
The same steps were taken on a Vultr VM instance and there's no issue which makes me suspect there's something different with the Digitalocean VM image for ubuntu 16.04.1. 

I am curious why this isn't playing nice still.

**Comment by sparklyballs** [comment]
I can't test that scenario as i don't use DO personally.

**Comment by msh100** [comment]
Happy to provide an instance to someone for a few days for debugging if necessary.

**Comment by msh100** [comment]
I had tried to setup openvpn-as manually on a DO droplet and encountered the same issue, I have then found this: https://www.digitalocean.com/community/questions/openvpn-access-server-not-working

For some reason Digitalocean does not like it being bound to the public IP and I have no idea why. Openvpn-as started working outside of a container when I changed it on `ovpn-init` to listen on all interfaces.

I have no idea if this is something you want to address within this container and would understand if not. Close at your own discretion.
